sympy/stats/tests/test_continuous_rv.py

```diff
@@ -543,6 +543,16 @@ def test_nakagami():
                                 (lowergamma(mu, mu*x**2/omega)/gamma(mu), x > 0),
                                 (0, True))
 
+def test_normal_inverse():
+    a, b = symbols('a b', positive=True)
+    z = Symbol('z')
+
+    X = NormalInverse('x', a, b)
+    assert density(X)(z) == sqrt(2)*sqrt(b)*sqrt(z**(-3))*exp(-b*(-a + z)**2/(2*a**2*z))/(2*sqrt(pi))
+    assert simplify(E(X)) == a
```

Comment:
Can the `simplify` be dropped here and below?

Reply:
Done